collector/container.go

```diff
@@ -0,0 +1,258 @@
+// returns containers metrics from HCS
+// https://github.com/Microsoft/hcsshim
+
+// +build windows
+
+package collector
+
+import (
+	"github.com/Microsoft/hcsshim"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/common/log"
+)
+
+func init() {
+	Factories["container"] = NewContainerMetricsCollector
+}
+
+// A ContainerMetricsCollector is a Prometheus collector for containers metrics
+type ContainerMetricsCollector struct {
+	// Presence
+	ContainerAvailable *prometheus.Desc
+	// memory
+	UsageCommitBytes            *prometheus.Desc
+	UsageCommitPeakBytes        *prometheus.Desc
+	UsagePrivateWorkingSetBytes *prometheus.Desc
+
+	// CPU
+	TotalRuntime100ns  *prometheus.Desc
+	RuntimeUser100ns   *prometheus.Desc
+	RuntimeKernel100ns *prometheus.Desc
```

Comment:
Since these three are scaled to seconds (right?), remove the 100ns suffix to avoid confusion

Reply:
corrected.